scripts/build_apple_frameworks.sh

```diff
@@ -148,10 +148,13 @@ for preset_index in "${!PRESETS[@]}"; do
     echo "Building preset ${preset} (${mode}) in ${preset_output_dir}..."
 
     # Do NOT add options here. Update the respective presets instead.
+    # Xcode multi-config presets leave CMAKE_BUILD_TYPE empty, so force EXECUTORCH_ENABLE_LOGGING per-mode.
     cmake -S "${SOURCE_ROOT_DIR}" \
           -B "${preset_output_dir}" \
           -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY="${preset_output_dir}" \
           -DCMAKE_BUILD_TYPE="${mode}" \
+          -UEXECUTORCH_ENABLE_LOGGING \
```

Comment:
is this needed?

Reply:
Yes - it gets enabled my macos.cmake present unconditionally due to pybind.cmake include, and we need to enabled logging for debug frameworks only.